src/Form/Nph/NphOrderForm.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Form\Nph;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type;
+use Symfony\Component\Validator\Constraints;
+
+class NphOrderForm extends AbstractType
+{
+    public static $urineColors = [
+        'Color 1' => 1,
+        'Color 2' => 2,
+        'Color 3' => 3,
+        'Color 4' => 4,
+        'Color 5' => 5,
+        'Color 6' => 6,
+        'Color 7' => 7,
+        'Color 8' => 8,
+    ];
+
+    public static $urineClarity = [
+        'Clean' => 'clean',
+        'Slightly Cloudy' => 'slightly_cloudy',
+        'Cloudy' => 'cloudy',
+        'Turbid' => 'turbid'
+    ];
+
+    public static $bowelMovements = [
+        'I was constipated (had difficulty passing stool), and my stool looks like Type 1 and/or 2' => 'difficult',
+        'I had diarrhea (watery stool), and my stool looks like Type 5, 6, and/or 7' => 'watery',
+        'I had normal formed stool, and my stool looks like Type 3 and/or 4' => 'normal'
+    ];
+
+    public static $bowelMovementQuality = [
+        'I tend to be constipated (have difficulty passing stool) - Type 1 and 2' => 'difficult',
+        'I tend to have diarrhea (watery stool) - Type 5, 6, and 7' => 'watery',
+        'I tend to have normal formed stool - Type 3 and 4' => 'normal'
+    ];
+
+    protected function addCollectedSampleFields($builder, $sample, $sampleLabel): void
```

Comment:
Please add type-hints to the parameters for the methods in this class.

Reply:
Done!